libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -417,6 +417,42 @@ void AP_DDS_Client::update_topic(geographic_msgs_msg_GeoPoseStamped& msg)
     }
 }
 
+void AP_DDS_Client::update_topic(sensor_msgs_msg_Imu& msg)
+{
+    update_topic(msg.header.stamp);
+    strcpy(msg.header.frame_id, BASE_LINK_NED_FRAME_ID);
+
+    auto &imu = AP::ins();
```

Comment:
Can you add the missing `#include <AP_IntertialSensor/AP_InertialSensor.h>`?

Reply:
the header has been added